packages/core-mobile/app/components/Fab.tsx

```diff
@@ -63,19 +58,8 @@ export const Fab: FC = () => {
         </View>
       ),
       onPress: () => {
-        let preSelectedToken
-        if (
-          [NetworkVMType.PVM, NetworkVMType.AVM].includes(
-            activeNetwork.vmName
-          ) &&
-          tokensWithBalance[0] !== undefined &&
-          tokensWithBalance[0].symbol === 'AVAX'
-        ) {
-          preSelectedToken = tokensWithBalance[0]
-        }
         navigation.navigate(AppNavigation.Wallet.SendTokens, {
-          screen: AppNavigation.Send.Send,
-          params: { token: preSelectedToken }
```

Comment:
The token preselecting logic has been moved to the components (SendPVM, SendAVM).